README.md

```diff
@@ -371,13 +371,15 @@ Examples:
 - `mac()`: Allows any valid MAC address
 
 ### Any - `any([validators])`
-Validates against a union of types. Use when a node can contain one of several types. It is valid
+Validates against a union of types. Use when a node **must** contain **one and only** of several types. It is valid
```

Comment:
Did this mean to say "one and only one"? I could also see this as one of the following:

Use when a node **must** contain **one** of several types.
Use when a node **must** contain **only one** of several types.

Reply:
Yes, yes it did. I shall fix.

Reply:
Fixed.

Question: Given that an empty any() allows anything, what happens if none of the specified